Library/Homebrew/cmd/doctor.rb

```diff
@@ -435,6 +443,7 @@ def check_access_usr_local
 
     You should probably change the ownership and permissions of /usr/local
     back to your user account.
+      sudo chown $(whoami):admin /usr/local
```

Comment:
Sure. If it's feasible to know in a time-effective way everything `brew` owns I'd be happy with that. The command would end up being something like:

```
sudo chown $(whoami):admin /usr/local && brew repair-permissions
```
On a permission borked El Cap install in that case, I'd presume?